app/services/dialog_import_service.rb

```diff
@@ -72,7 +79,6 @@ def create_import_file_upload(file_contents)
 
   def import_from_dialogs(dialogs)
     raise ParsedNonDialogYamlError if dialogs.empty?
-
```

Comment:
Nit, but can you restore this whitespace if it was otherwise untouched in this PR?